### PR TITLE
added test code to check of flake8

### PR DESCRIPTION
**Pull request by himaaaatti**
I Added code to Makefile of link target, and adapt each test file.
if you don't check test code by flake purposely, you don't have to merge this request.

### CODE REVIEW

File: test/configs/syntaxerr.py [hunk 1]
```diff
@@ -21,4 +21,4 @@
 # It is really dumb that we need a giant copyright notice for this one line
 # file. Fucking lawyers :(
 
-sdf [
```
Comment (flacjacket) [inline test/configs/syntaxerr.py]: You should make this `sdf [  # noqa`, we want to test something that is incorrect

File: test/test_command.py [hunk 2]
```diff
@@ -246,7 +246,7 @@ def test_items_group(self):
     g = self.c.group
     assert g.items("layout") == (True, [0, 1, 2])
 
-    win = self.testWindow("test")
+#      win = self.testWindow("test")
```
Comment (flacjacket) [inline test/test_command.py]: All of these in this file need to be run

File: test/test_fakescreen.py [hunk 3]
```diff
@@ -234,7 +234,7 @@ def test_float_first_on_second_screen(self):
 
 @Xephyr(False, FakeScreenConfig(), two_screens=False, width=900, height=980)
 def test_float_change_screens(self):
-    #add some eyes, and float clock
+    "add some eyes, and float clock"
```
Comment (flacjacket) [inline test/test_fakescreen.py]: `# add some eyes, and float clock`

File: test/test_fakescreen.py [hunk 4]
```diff
@@ -247,7 +247,7 @@ def test_float_change_screens(self):
     assert self.c.window.info()['y'] == 0
     assert self.c.window.info()['group'] == 'a'
 
-    #put on group b
+#      put on group b
```
Comment (flacjacket) [inline test/test_fakescreen.py]: `    # put on group b`

File: test/test_fakescreen.py [hunk 5]
```diff
@@ -370,15 +370,15 @@ def test_hammer_tile(self):
...
-        old_group = (i + 1) % 4
-        if old_group == 0:
-            name = 'a'
-        elif old_group == 1:
-            name = 'b'
-        elif old_group == 2:
-            name = 'c'
-        elif old_group == 3:
-            name = 'd'
+        # old_group = (i + 1) % 4
```
Comment (flacjacket) [inline test/test_fakescreen.py]: If some variable is not used, that doesn't mean you should comment it out, I'm not going to point them all out, but they all should be uncommented

**Comment by himaaaatti** [comment]
Thank you many comment. I will fix some code that you pointed for the latest develop branch, and I send again new PR.

**Pull request closed without merging**